CHANGELOG.md

```diff
@@ -73,7 +73,7 @@ This release supports the following [libp2p protocols](https://docs.libp2p.io/co
 | Protocol | Spec status | Protocol id |
 | ---: | :---: | :--- |
 | [`17/WAKU-RLN`](https://rfc.vac.dev/spec/17/) | `raw` | `/vac/waku/waku-rln-relay/2.0.0-alpha1` |
-| [`18/WAKU2-SWAP`](https://rfc.vac.dev/spec/18/) | `raw` | `/vac/waku/swap/2.0.0-alpha1` |
+| [`18/WAKU2-SWAP`](https://rfc.vac.dev/spec/18/) | `raw` | `/vac/waku/swap/2.0.0-beta1` |
```

Comment:
Actually this is a previous release, don't want to edit this part of the CHANGELOG. Can add to new work in progress changelog